src/platform/tls_stub.c

```diff
@@ -1877,8 +1877,10 @@ QuicEncrypt(
     UNREFERENCED_PARAMETER(AuthDataLength);
     UNREFERENCED_PARAMETER(AuthData);
     uint16_t PlainTextLength = BufferLength - QUIC_ENCRYPTION_OVERHEAD;
-    ((uint64_t*)(Buffer + PlainTextLength))[0] = Key->Secret;
-    ((uint64_t*)(Buffer + PlainTextLength))[1] = 0;
+    uint64_t WriteVal = Key->Secret;
+    QuicCopyMemory(Buffer + PlainTextLength, &WriteVal, sizeof(WriteVal));
```

Comment:
You shouldn't need a temporary variable.

Reply:
I needed the temp for the 2nd value. But if I use zero, I can get rid of it.